### PR TITLE
Fix #4648

### LINKED ISSUE #4648: make user send the CS UI task URL when they ask for help
Currently users send the dashboard URL, or (more rarely) the uploadlog URL, hardly a task name.
I'd rather have them always point to the CS UI, from where dashboard, config, webdir, log are all easily available (and all in all is what Emilis developed it for).
Before I start advertising for this, I thought it will be good to print this URL to user screen.

@mmascher @emaszs @dciangot what do you think of proposal in following screenshot ?

screenshot from 2016-10-28 11 15 43

### PR DESCRIPTION
**Pull request by belforte**

### CODE REVIEW

File: src/python/CRABClient/Commands/status.py [hunk 1]
```diff
@@ -121,10 +121,17 @@ def printShort(self, dictresult, username):
...
 
-        ## Show dashboard URL for the task.
+        ## Show server and dashboard URL for the task.
+
+        taskname = urllib.quote(self.cachedinfo['RequestName'])
+        ## CRAB Server UI URL for this task is always useful
+        crabServerUIURL = "https://cmsweb.cern.ch/crabserver/ui/task/" + taskname
+        msg = "%sTask URL to use for HELP:\t%s%s" % (colors.GREEN, crabServerUIURL, colors.NORMAL)
+
+	self.logger.info(msg)
```
Comment (emaszs) [inline src/python/CRABClient/Commands/status.py]: This needs an extra space for correct indentation

File: src/python/CRABClient/Commands/status2.py [hunk 2]
```diff
@@ -163,10 +163,16 @@ def printTaskInfo(self, crabDBInfo, username):
                 msg += "%s" % (status)
         self.logger.info(msg)
 
-        # Show dashboard URL for the task.
+        # Show server and dashboard URL for the task.
+	taskname = urllib.quote(self.cachedinfo['RequestName'])
+
+	## CRAB Server UI URL for this task is always useful
+        crabServerUIURL = "https://cmsweb.cern.ch/crabserver/ui/task/" + taskname
```
Comment (emaszs) [inline src/python/CRABClient/Commands/status2.py]: This has one space too many

**Comment by belforte** [comment]
I do not like tabs either, indeed took a wihile to get the proper number of them :-( But I used them because the nearby code does, so I thought consistency was useful. I know it is not your code, it is not mine either. But changing all prints in this to use spaces may be too much work. Maybe we only do it in status2 ?

**Comment by emaszs** [comment]
What editor are you using? In status.py, I only see two tabs, one in line 131 and another one in line 338, the second one was there for a while so I guess it's fine. In status2.py, I only see 5 tabs coming from your commit on lines 167-174.

**Comment by emaszs** [comment]
Ah, a misunderstanding, I meant the tabs used to indent code, not the tabs used in print statements (these are fine of course). Tabs used to indent code can cause some truly mysterious errors.

**Comment by belforte** [comment]
ah.. right. Sorry. I had started asking if you meant indentation in the code or in the status output in reply to the deleted comment.. then got sidetracked.
Sure, better use spaces in python. I'll fix this. Thanks for spotting.

**Pull request merged**